### DISCUSSION TITLE
Explore page redesign?

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#30131] [Mobile]
What do you think about my redesign and which one would you prefer?

Original:
> ![Screenshot_20220819-234612.png](https://user-images.githubusercontent.com/94877880/185741622-d3b03463-5237-49d2-89f9-37b409861b86.png)

Redesigned:
> ![GitHub redesign.jpg](https://user-images.githubusercontent.com/94877880/185741566-95078710-4cfa-4a93-9c66-0dfc56b1de32.jpg)

I actually made this to consume less screen space.

**Comment by radar-blip** [comment]
Original
